ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayAnnotation/DisplayAnnotation.qml

```diff
@@ -0,0 +1,60 @@
+// [WriteFile Name=DisplayAnnotation, Category=Layers]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.8
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            BasemapLightGrayCanvas {}
+
+            FeatureLayer {
+                ServiceFeatureTable {
+                    id: featureTable
+                    url: "https://services1.arcgis.com/6677msI40mnLuuLr/arcgis/rest/services/East_Lothian_Rivers/FeatureServer/0"
+                }
+            }
+
+            AnnotationLayer {
+                ServiceFeatureTable {
```

Comment:
This code differs from the C++ implementation. C++ constructs via URL and QML constructs via a ServiceFeatureTable. We should match whatever the other SDKs are doing. Looks like on Android, the feature layer is constructed with a feature table and the annotation layer is constructed with a URL - https://github.com/Esri/arcgis-runtime-samples-android/blob/master/kotlin/display-annotation/src/main/java/com/esri/arcgisruntime/sample/displayannotation/MainActivity.kt#L38-L43

Reply:
sounds like a bug. marking this resolved

Reply:
This will be fixed with 100.8 build 2702. I would recommend updating the sample to match the same workflow as C++ and other SDKs for consistency.